src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/LineWrappingHandler.java

```diff
@@ -135,7 +134,11 @@ public void checkIndentation(DetailAST firstNode, DetailAST lastNode, int indent
                             getNextNodeLine(firstNodesOnLines, node),
                             true
                         );
-                    checkAnnotationIndentation(atNode, annotationLines, indentLevel);
+                    checkAnnotationIndentation(
+                        atNode,
+                        annotationLines,
+                        getLineStart(atNode),
```

Comment:
What if the parameter `startIndent` isn't -1? Shouldn't you switch between it's value too?
See line 153, https://github.com/checkstyle/checkstyle/pull/4936/files#diff-1ab987049997f8605718e00e9fe92d25R153 .